code/__DEFINES/~skyrat_defines/DNA.dm

```diff
@@ -128,6 +128,7 @@
 #define SPECIES_INSECT "insect"
 #define SPECIES_MAMMAL "mammal"
 #define SPECIES_PODPERSON_WEAK "podweak"
+#define SPECIES_XIARN "SPECIES_XIARN" // BUBBER CHANGE, xiarn
```

Comment:
```suggestion
#define SPECIES_XIARN "xiarn" // BUBBER CHANGE, xiarn
```
If you are going to use define you should actually have it have a proper word replacement

Reply:
resolved this already right before you made the comment